django/contrib/admin/templatetags/log.py

```diff
@@ -5,7 +5,9 @@
 
 class AdminLogNode(template.Node):
     def __init__(self, limit, varname, user):
-        self.limit, self.varname, self.user = limit, varname, user
+        self.limit = limit
+        self.varname = varname
+        self.user = user
```

Comment:
~~Was this change intentional?~~ 🤔 

(I agree this is easier to grok but not sure what the standard is here)

Ah I see other places use this format so at least it's consistent.

Reply:
Sorry I didn't hit the button to post the above comment - yes it's intentional, see the comment for details.

It could be put into a separate commit removing this pattern from everywhere.

Reply:
I'd put it the other way round: why leave in a slow pattern that can be copied in the future? IMO it doesn't clarify the code, it actually takes more mental load to figure out where variables are assigned.

Reply:
For me, it's just an extra noise in the history. We try to avoid trivial code reformatting unless it's a big problem.

Reply:
Okay, fair enough.